spec/helpers.lua

```diff
@@ -29,6 +29,10 @@ local client = reload_module("spec.internal.client")
 local wait = reload_module("spec.internal.wait")
 
 
+-- redo the patches to make kong PDK available to timerng timers
```

Comment:
Could you explain the context of "make kong PDK available to timerng timers", I don't quite get it.